src/ledger/meta.zig

```diff
@@ -359,6 +367,8 @@ pub const MerkleRootMeta = struct {
             // shred that contains a proper merkle root would constitute
             // a valid duplicate shred proof.
             .merkle_root = shred.fields.merkleRoot() catch null,
+            // QUESTION: This does not look like the _first_ recieved shred index,
+            // but the index of _this_ shred we used to construct the MerkleRootMeta
```

Comment:
That's a great point. This code is actually correct because it is only used for the first shred. But it's not clear from the naming.
41878cb670dd4837ddbb722172ff3d45488c651c